torch/linalg/__init__.py

```diff
@@ -939,17 +941,15 @@
 symmetric if real, but this is not checked internally. Instead, just the lower
 triangular part of the matrix is used in the computations.
 
-If :attr:`tol` is not specified and :attr:`A` is a matrix of dimensions `(m, n)`,
-the tolerance is set to be
+If :attr:`rtol` is not specified and :attr:`A` is a matrix of dimensions `(m, n)`,
+the relative tolerance is set to be
 
 .. math::
 
-    \text{tol} = \sigma_1 \max(m, n) \varepsilon
+    \text{rtol} = \max(m, n) \varepsilon
 
-where :math:`\sigma_1` is the largest singular value
-(or eigenvalue in absolute value when :attr:`hermitian`\ `= True`), and
-:math:`\varepsilon` is the epsilon value for the dtype of :attr:`A` (see :class:`torch.finfo`).
-If :attr:`A` is a batch of matrices, :attr:`tol` is computed this way for every element of
+and :math:`\varepsilon` is the epsilon value for the dtype of :attr:`A` (see :class:`torch.finfo`).
+If :attr:`A` is a batch of matrices, :attr:`rtol` is computed this way for every element of
```

Comment:
Even though it was like this in the previous version, I believe it's safe to delete it here and in the docs of `pinv`, as this is the expected behaviour really.